Student/StudentUITests/Element/AudioRecorder.swift

```diff
@@ -15,9 +15,25 @@
 //
 
 import Foundation
+import SwiftUITest
 
-enum AudioRecorder: String, Element {
+private enum AudioRecorder: String, ElementId {
     case cancelButton, clearButton, recordButton, sendButton, stopButton, timeLabel
     // embedded AudioPlayer
     case currentTimeLabel, loadingView, playPauseButton, remainingTimeLabel, timeSlider
 }
+
+enum AudioRecorderElements {
+
+    static var cancelButton = app.find(AudioRecorder.cancelButton)
+    static var clearButton = app.find(AudioRecorder.clearButton)
+    static var recordButton = app.find(AudioRecorder.recordButton)
+    static var sendButton = app.find(AudioRecorder.sendButton)
+    static var stopButton = app.find(AudioRecorder.stopButton)
+    static var timeLabel = app.find(AudioRecorder.timeLabel)
+    static var currentTimeLabel = app.find(AudioRecorder.currentTimeLabel)
+    static var loadingView = app.find(AudioRecorder.loadingView)
+    static var playPauseButton = app.find(AudioRecorder.playPauseButton)
+    static var remainingTimeLabel = app.find(AudioRecorder.remainingTimeLabel)
+    static var timeSlider = app.find(AudioRecorder.timeSlider)
```

Comment:
if these are lazy initialized in a test, would other tests that use them end up referencing the wrong XCTestCase?

Reply:
Thanks. I pushed a fix to always recompute the app variable.